tap-gui/plugins/runtime-resource-visibility.hbs.md

```diff
@@ -12,6 +12,8 @@ Do one of the following actions to access the Runtime Resources Visibility plug-
 - [Install the Tanzu Application Platform Full or View profile](../../install-intro.md)
 - [Install Tanzu Application Platform without using a profile](../../install-intro.md) and then
   install [Tanzu Application Platform GUI separately](../install-tap-gui.md)
+- If you're not using any metric server on your cluster, make sure to set the
+`skipMetricsLookup` to `true` on the `app.config.yaml` file of your application.
```

Comment:
I thought that the `app.config.yaml` is not configured per-application, but per installation? If it's the latter, it seems like this would be an instruction for administrators, not typical users.

Reply:
That's a good question. Maybe Chinigo would know better?

Reply:
Addressed, can you please re-review it? @menehune23 @chinigo

Reply:
Addressed, please consider resolved.